src/juicebox/tools/clt/old/PreProcessing.java

```diff
@@ -58,6 +58,7 @@ public PreProcessing() {
                 + "           : -g <graphs file> Add the text graphs file to the Hi-C file header\n"
                 + "           : -n Don't normalize the matrices\n"
                 + "           : -z <double> scale factor for hic file\n"
+                + "           : -a <1, 2, 3> filter based on inner, outer, tandem alignment\n"
```

Comment:
if you can add the word "or" before tandem.
```suggestion
                + "           : -a <1, 2, 3> filter based on inner, outer, or tandem alignment\n"
```